jwst/ami/lg_model.py

```diff
@@ -79,7 +79,7 @@ def __init__(
         )
         self.ctrs = self.mask.ctrs
         self.d = self.mask.hdia
-        self.D = self.mask.active_D
+        self.active_D = self.mask.active_D
```

Comment:
having attributes named `d` and `D` was causing a "blocker"-level code smell failure in sonarqube